solutions/src/main/java/ru/sberbank/school/task06/CountMapImpl.java

```diff
@@ -0,0 +1,117 @@
+package ru.sberbank.school.task06;
+
+import ru.sberbank.school.task06.Player;
+
+import java.time.Period;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Интерфейс мапы, которая ведет подсчёт вхождений элемента в контейнер.
+ *
```

Comment:
Empty line should be followed by &lt;p&gt; tag on the next line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a938354026>*